rules/judgment.tex

```diff
@@ -80,7 +82,17 @@ \subsection{Referee--Team Communication}
 During all phases of the match teams and referees are communicating with respect to each other.
 
 \subsection{Referees During the Match}
+\label{sec:referee_during_match}
 
 The head referee and the assistant referees should wear socks \emph{of black or dark blue color} and avoid reserved colors (white and green) in their leg clothing.
 They may enter the field in particular situations, \eg, to remove a robot when applying a penalty.
 They should avoid interfering with the robots as much as possible.
+
+\subsection{Referee List For Friendly Games}
+\label{sec:referee_list}
+
+During the RoboCup competition setup days and during the game days, several teams may want to participate in friendly games with each other.
+In this case, there should be a list of all the head and assistant referees that are willing to judge a game.
+The referees on the list should be voluntary but participation should be encouraged.
```

Comment:
Maybe specify this is offer to teams, but they can still choose their referees for there friendly games.

Reply:
Maybe also say that new referees seeking experience are particularly welcome? IIRC this was one of the intentions behind this.